app/views/schedules/_event.html.haml

```diff
@@ -1,9 +1,15 @@
-.panel.panel-default.event-panel{ id: "link-#{event.id}" }
+.panel.panel-default.event-panel{ onClick: 'eventClicked(event, this);', "data-url" => "#{url_for(conference_program_proposal_path(@conference.short_title, event.id))}" }
   .panel-body
     - if speaker = event.speakers.first
       = image_tag speaker.gravatar_url, :class => "img-circle pull-right all-speaker-pic", |
                                         :alt => speaker.name, |
                                         :title => speaker.name |
+    - if  current_user && !@favourites
```

Comment:
I think we should show the star in favorites too, so that it is easy for the attendee to unstar a talk.